lib/src/__tests__/DatePickerTestingLib.test.tsx

```diff
@@ -21,4 +22,56 @@ describe('<DatePicker />', () => {
 
     expect(screen.getByRole('button', { name: '2010' })).toBeDisabled();
   });
+
+  it("Doesn't close picker on selection in Mobile mode", () => {
```

Comment:
We have been including "it" as part of the sentence

```suggestion
  it("doesn't close picker on selection in Mobile mode", () => {
```

Reply:
For now, all the tests are written with first capital letter, so we need to change or everything or nothing

Reply:
If we assume that we have 2000 tests on material-ui, 10 tests on material-ui-x and 100 tests on material-ui-pickers, 5% of the tests are written with a capital first letter. The closest we get to 0%, the lower entropy, the. better :)